pkg/apis/serving/v1alpha1/kfservice_defaults.go

```diff
@@ -7,9 +7,9 @@ import (
 
 // Default Values
 var (
-	DefaultTensorflowVersion  = "1.12"
-	DefaultXGBoostVersion     = "1.12"
-	DefaultScikitLearnVersion = "1.12"
+	DefaultTensorflowServingVersion  = "1.13.0"
```

Comment:
can we make these DefaultServingVersion as flags of kfserving main?
```
flag.StringVar(&DefaultTensorflowServingVersion , "DefaultTensorflowServingVersion", "1.13.0", "default tensorflow serving version.")
```

Reply:
See https://github.com/kubeflow/kfserving/issues/68, feel free to help out :)

Reply:
Where do these flags get read from? Is this the standard way to read from configmaps?